<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trades</class>
 <widget class="QMainWindow" name="trades">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_trades">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1051</width>
      <height>470</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_active_trades">
     <attribute name="title">
      <string>Active Trades</string>
     </attribute>
     <widget class="QTableWidget" name="tbl_active_trades">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1031</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>No. Trade</string>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Entry Date</string>
       </property>
       <property name="toolTip">
        <string extracomment="Hola"/>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Symbol</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Direction</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Entry Price</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Position Size</string>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Notional Value</string>
       </property>
       <property name="toolTip">
        <string extracomment="Total De Monedas Compradas">The total value of your position.</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stop Loss</string>
       </property>
       <property name="toolTip">
        <string>The price at which you will accept the losses and exit the trade.
The stop loss price should be defined before the trade starts. 
It determines how much you will lose if the market moves against your position.</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Take Profit</string>
       </property>
       <property name="toolTip">
        <string>Take profit prices represent your targets. They should be defined before the trade starts.
Some traders set multiple take profit targets and calculate the average exit price after the operation is done</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Current Price</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fees</string>
       </property>
       <property name="toolTip">
        <string>0.1% trading fee multiplied by 2 (entry and exit)</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>P&amp;L</string>
       </property>
       <property name="toolTip">
        <string>P&amp;L or PnL represents the profit and losses for each trade.

PnL (long positions) 
= &quot;(Exit price * position size) - Notional Value - fees&quot;
= ($19,214 * 0.057) - $1,019.16 - $2.04
= $74

PnL (short positions)
= &quot;Notional Value - (Exit price * position size) - fees&quot;
= $675.50 - ($18,354 * 0.035) - $1.35
= $31.76</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>P&amp;L%</string>
       </property>
       <property name="toolTip">
        <string>The % of gains or losses in relation to the notional value. 
In spot markets, the notional value represents your initial investment. But in leveraged trading, the notional value can be much higher than the initial investment.</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_update">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>390</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_end_trade">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>390</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>End Trade</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_pnl">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>410</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>pnl</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_pnlp">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>410</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>pnlp</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_notional_value">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>notional_value</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_notional_value_title">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Notional Value</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_pnl_title">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>PnL</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_pnlp_title">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>390</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>PnL%</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_historic">
     <attribute name="title">
      <string>Historic</string>
     </attribute>
     <widget class="QTableWidget" name="tbl_historic_trades">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1031</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>No. Trade</string>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Entry Date</string>
       </property>
       <property name="toolTip">
        <string extracomment="Hola"/>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Exit Date</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Symbol</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Direction</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Entry Price</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Position Size</string>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Notional Value</string>
       </property>
       <property name="toolTip">
        <string extracomment="Total De Monedas Compradas">The total value of your position.</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stop Loss</string>
       </property>
       <property name="toolTip">
        <string>The price at which you will accept the losses and exit the trade.
The stop loss price should be defined before the trade starts. 
It determines how much you will lose if the market moves against your position.</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Take Profit</string>
       </property>
       <property name="toolTip">
        <string>Take profit prices represent your targets. They should be defined before the trade starts.
Some traders set multiple take profit targets and calculate the average exit price after the operation is done</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Exit Price</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fees</string>
       </property>
       <property name="toolTip">
        <string>0.1% trading fee multiplied by 2 (entry and exit)</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>P&amp;L</string>
       </property>
       <property name="toolTip">
        <string>P&amp;L or PnL represents the profit and losses for each trade.

PnL (long positions) 
= &quot;(Exit price * position size) - Notional Value - fees&quot;
= ($19,214 * 0.057) - $1,019.16 - $2.04
= $74

PnL (short positions)
= &quot;Notional Value - (Exit price * position size) - fees&quot;
= $675.50 - ($18,354 * 0.035) - $1.35
= $31.76</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>P&amp;L%</string>
       </property>
       <property name="toolTip">
        <string>The % of gains or losses in relation to the notional value. 
In spot markets, the notional value represents your initial investment. But in leveraged trading, the notional value can be much higher than the initial investment.</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_AddTrade">
     <attribute name="title">
      <string>Add Trade</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>931</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="layout_add_trade">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="rowWrapPolicy">
        <enum>QFormLayout::DontWrapRows</enum>
       </property>
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_entry_date">
         <property name="text">
          <string>Entry Date:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="date_entry_date"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_symbol">
         <property name="text">
          <string>Symbol:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txt_symbol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_direction">
         <property name="text">
          <string>Direction:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cmb_direction">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="maxVisibleItems">
          <number>2</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl_entry_price">
         <property name="text">
          <string>Entry Price:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txt_entry_price">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbl_position_size">
         <property name="text">
          <string>Position Size:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="txt_position_size">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lbl_stop_loss">
         <property name="text">
          <string>Stop Loss:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="txt_stop_loss">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lbl_take_profit">
         <property name="text">
          <string>Take Profit:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="txt_take_profit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="btn_add_trade">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_salir">
    <property name="geometry">
     <rect>
      <x>985</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_patch_notes">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Patch Notes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
